### ISSUE TITLE
git clone results in fatal: bad config line 5 in file .git/config

### ISSUE THREAD

**Issue by vaskas** [berryamin/berrymuch#18]
## Steps to reproduce
1. Try to clone a repo a few times, e.g.:
  ```
  git clone https://git.suckless.org/ii
  ```
2. See the operation fail most times with:
  ```
  fatal: bad config line 5 in file .git/config
  ```

## Expected result
The repository should be cloned

## Work around
Keep re-trying. Try not to smash your phone if it's still not working.

**Comment by vaskas** [comment]
@yamsergey the new version (2.24.1) doesn't fix this apparently

**Comment by berryamin** [comment]
obviously linked to the fact that QNX is an RTOS ...
http://www.qnx.com/developers/docs/6.4.1/neutrino/getting_started/s1_inter.html

**Comment by yamsergey** [comment]
🤔  means we should adjust git sources to disable interrupt while it's checking out?

**Comment by berryamin** [comment]
Yep, sort of .. don't know the exact details of what we should do, but sounds like implementing the first patches git on a real time os :)
Would behave like "trap" does at shell level when receiving a sigkill signal for example ...

**Comment by berryamin** [comment]
I had made some tries to fix this without success:

https://github.com/berryamin/berrymuch/pull/22/files#diff-3e9ed3f4fb01f5a49fd35c6530571d20

**Comment by yamsergey** [comment]
I also did some investigation quite long ago. https://github.com/mordak/playbook-dev-tools/issues/7#issuecomment-425018434 , not a lot but maybe a way to reproduce

**Comment by karawitan** [comment]
Git 2.50 will be available soon as part of qnx-packages suite from @sw7ft . Work is in progress to integrate the qnx-packages suite with berrymuch in an elegant way.

[Git 2.50 for QNX announcement on Patreon](https://www.patreon.com/posts/git-2-50-for-qnx-132240507)
